RealestateApi/Controllers/RealEstateController.cs

```diff
@@ -52,6 +55,13 @@ public async Task<ActionResult<List<RealEstateRequestDto>>> GetAllRealEstate()
         [HttpPost(Name = "AddRealEstate")]
         public async Task<ActionResult<RealEstateModel>> AddRealEstateAsync(AddRealEstateRequestDto realEstateDto)
         {
+            var validationResult = _realEstateRequestValidatorDto.Validate(realEstateDto);
+
+            if (!validationResult.IsValid)
```

Comment:
I suspect that if you remove this If statement, and line 58, the validator will be triggered automatically.
Can you give it a try and let me know please?